1-Draft/RFC00XX-Native-Command-Error-Handling.md

```diff
@@ -0,0 +1,229 @@
+---
+RFC: RFC00XX
+Author: Jason Helmick
+Status: Draft
+Area: Core
+Comments Due: 10/31/2020
+---
+
+# Native Command Error Handling
+
+PowerShell scripts using native commands would benefit from being able to use error handling
+features like those used by cmdlets.
+
+## Motivation
+
+In PowerShell by default, script processing continues when non-terminating errors occur. This is a
+benefit when expecting non-terminating errors in normal execution such as non-responsive computers from
+a list. This default behavior is controlled with the preference variable
+`$ErrorActionPreference` default of `Continue`.
+
+During development and debugging, often customers prefer that script execution stops when a
+non-terminating error occurs. PowerShell currently supports customers with this ability by setting
+the preference variable in the script.
+
+```Powershell
+$ErrorActionPreference = 'Stop'
+```
+
+Native commands return an exit code to the calling application which will be zero for success or
```

Comment:
```suggestion
Native commands usually return an exit code to the calling application which will be zero for success or
```

Reply:
I would put the `usually` in `which will usually be zero for success`. I think every process must report an exit code